plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizardTest.java

```diff
@@ -16,16 +16,22 @@ public void setUp() {
     } catch (NullPointerException ex) {
       wizard = new StandardProjectWizard();
     }
+    wizard.init(null, null);
```

Comment:
seems to be needed:

Running com.google.cloud.tools.eclipse.appengine.newproject.StandardProjectWizardTest
Tests run: 3, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 0.202 sec <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.newproject.StandardProjectWizardTest
testOnePage(com.google.cloud.tools.eclipse.appengine.newproject.StandardProjectWizardTest)  Time elapsed: 0.136 sec  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.google.cloud.tools.eclipse.appengine.newproject.StandardProjectWizardTest.testOnePage(StandardProjectWizardTest.java:34)